scripts/3.TCGA-MLexample_Pathway.py

```diff
@@ -87,39 +87,45 @@
 
 # In[7]:
 
+# Perform 5 fold cross validation
+folds = 5
```

Comment:
Noting that previously 3 folds were used (default). This changes the number of folds to 5, which will result in better performance at the cost of additional computation.